packages/@aws-cdk/aws-events/lib/schedule.ts

```diff
@@ -42,6 +42,8 @@ export abstract class Schedule {
       throw new Error('Cannot supply both \'day\' and \'weekDay\', use at most one');
     }
 
+    const minuteUndefinedWarning = options.minute ?? "When 'minute' is undefined in CronOptions, '*' is used as the default value, scheduling the event for every minute within the supplied parameters.";
```

Comment:
```suggestion
    const minuteUndefinedWarning = options.minute ? "When 'minute' is undefined in CronOptions, '*' is used as the default value, scheduling the event for every minute within the supplied parameters." : undefined;
```

`a ?? b` means: `a if it has a value, otherwise b`.

Reply:
I like that message better. I'll go with that suggestion.